doc/content/getting-started/aws/ecs/database-operations/_index.md

```diff
@@ -124,11 +128,28 @@ tti-lw-stack,is-db,create-oauth-client,--tenant-id=NULL,--id=console-eu1,--name=
 {{</ tabs/container >}}
 
 > **Notes:** 
->- Replace `--tenant-id=NULL` with `--tenant-id=your-tenant-id` in single-tenant deployments.
+>- Replace `--tenant-id=NULL` with `--tenant-id=your-tenant-id` in single-tenant deployments. `id` and `secret` can be found in the `*-console-oauth-client` secret in the AWS Secrets Manager.
```

Comment:
Can you use the full path; something like `<Network>/<cluster>/.../console-oauth-client`